calcom/synclinear.com
# Allow syncing projects to milestones

# Summary

- Adds an option to the UI
- Allows syncing Linear Project to GitHub Milestone
  - When a Ticket is added/removed from a Project, the synced Issue is added/removed from the corresponding Milestone

<img width="1070" alt="image" src="https://github.com/calcom/synclinear.com/assets/36117635/9f5b7d6a-101f-4df4-aaf9-26c744469cfd">

<img width="591" alt="image" src="https://github.com/calcom/synclinear.com/assets/36117635/eb18451c-f330-46c2-bf91-98b8528eb9cd">

## Test Plan

- Go to UI and enable Project sync. It should show a loading state then finish without error.
- In a synced Linear team, add a synced Ticket to a Project. Its corresponding Issue should be added to a newly-created Milestone of the same name.
- Repeat step 2 with a new Ticket. The corresponding Issue should be added to the same Milestone.

## Related Issues

n/a
